### PR TITLE
Fix comments

### PR DESCRIPTION
**Pull request by 01vadim10**
My description

### CODE REVIEW

**Comment by prokharovaky** [comment]
Pull Request after removing unnecessary comments and other changes

File: SneakPeek/Controllers/MoviesController.cs [hunk 1]
```diff
@@ -8,24 +8,21 @@
 using Newtonsoft.Json.Linq;
 namespace SneakPeek.Controllers
 {
```
Comment (01vadim10) [inline SneakPeek/Controllers/MoviesController.cs:10]: Can we make it file-scoped?

File: SneakPeek/Controllers/MoviesController.cs [hunk 2]
```diff
@@ -35,26 +32,19 @@ public async Task<IActionResult> Get()
...
-                    return Content(jsonResult, "application/json"); // Return string in json format
+                    return Content(jsonResult, "application/json; charset=utf-8"); 
                 } 
-
-
             }
             catch (IOException ex)
             {
-                return StatusCode(500, $"Internal server error: {ex.Message}");
+                return StatusCode(500,$"Internal server error: {ex.Message}");
```
Comment (01vadim10) [inline SneakPeek/Controllers/MoviesController.cs:42]: Whitespace?

File: SneakPeek/TestClass.cs [hunk 3]
```diff
@@ -0,0 +1,5 @@
+﻿namespace SneakPeek;
+
+public class TestClass
+{
+}
```
Comment (01vadim10) [inline SneakPeek/TestClass.cs:5]: Please, remove redundant class

File: SneakPeekExample.Tests/SneakPeekExample.Tests.csproj [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```
Comment (01vadim10) [inline SneakPeekExample.Tests/SneakPeekExample.Tests.csproj:3]: Please, remove this project

File: SneakPeak.IntegrationTests/MoviesControllerTests.cs [hunk 5]
```diff
@@ -1,6 +1,7 @@
 using Microsoft.VisualStudio.TestPlatform.TestHost;
 using Microsoft.AspNetCore.Mvc.Testing;
 using FluentAssertions;
+using System.Net;
 
 namespace SneakPeak.Tests
 {
```
Comment (01vadim10) [inline SneakPeak.IntegrationTests/MoviesControllerTests.cs:7]: Can we make it file-scoped?

File: SneakPeek/Models/Movie.cs [hunk 6]
```diff
@@ -10,5 +10,4 @@ public class Movie
         public List<string> director { get; set; }
         public double rating { get; set; }
     }
```
Comment (01vadim10) [inline SneakPeek/Models/Movie.cs:12]: Can we make it file-scoped?

File: SneakPeek/Controllers/MoviesController.cs (hunk 1 above)
Comment (01vadim10) [reply SneakPeek/Controllers/MoviesController.cs:11]: Won't be fixed because it is a stupid idea

**Pull request merged**